src/psyclone/psyGen.py

```diff
@@ -5335,6 +5393,58 @@ def else_body(self):
             return self._children[2]
         return None
 
+    @staticmethod
+    def create(if_condition, if_body, else_body=None):
+        '''Create an IfBlock instance given valid instances of an
+        if_condition, an if_body and an optional else_body.
+
+        :param if_condition: the PSyIR node containing the if \
+            condition of the if block.
+        :type if_condition: :py:class:`psyclone.psyGen.Node`
+        :param if_body: the PSyIR node containing the if body of \
```

Comment:
`if_body` and `else_body` docstrings do not match the implementation, you are expecting a list of Nodes

Reply:
Well spotted.

Reply:
docstring types are good now but I would still change the descriptions to "the PSyIR nodes representing the ..." in both the `if_body and `else_body`

Reply:
Done.